plugins/modules/zos_operator.py

```diff
@@ -35,10 +35,28 @@
     default: false
   debug:
     description:
-      - Return debugging information.
+      - Return rexx debugging information.
     type: bool
     required: false
     default: false
+  security:
```

Comment:
I matched the original intent from oper, where -v enables ISFSECTRACE='ON', which adds security trace/decision info to the output stream (stdout, not seclog).  As I'm looking at it, I'm going to re-trace that, because I believe I flipped -s and -v.  From the opercmd: -v: print out verbose security information... I've used -v for security, and -s for verbose... it's wierd, but that's what's there.

Reply:
Wait actually currently attaches whether or not verbose is selected (it's set if blank to (VERBOSE).  Not hard to make it optional.  So, to clarify on "merging"... you want one option: Verbose, which means "verbose -v plus -s option".  That simplifies things.  What about debug -d option, which I copied from opercmd... it really just dumps some trace data from the rexx script.  Maybe drop that altogether?  I can have it either way this afternoon, so let me know which is preferred.

Reply:
Just recording our prior discussion, merging makes sense as we both agreed and we also decided that `-d` which is REXX debug on a script we control adds little value to enabled and expose to the users.

Reply:
Looks good, you merged logging under one verbose, thank you.